grape/graph/id_indexer_impl.h

```diff
@@ -0,0 +1,1164 @@
+/** Copyright 2020 Alibaba Group Holding Limited.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#ifndef GRAPE_GRAPH_ID_INDEXER_IMPL_H_
+#define GRAPE_GRAPH_ID_INDEXER_IMPL_H_
+
+#include <vector>
+
+namespace grape {
+
+namespace id_indexer_impl {
+
+struct prime_number_hash_policy {
```

Comment:
Can we use

```
using prime_number_hash_policy = ska:: prime_number_hash_policy;
```

rather than copying the code there?

Reply:
done